src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -107,29 +108,31 @@ export default Backbone.Model.extend({
     };
   },
   getEntityFilter() {
+    const states = this.currentOrg.getStates();
     const filtersState = this.getFilters();
     const clinicianId = this.get('clinicianId');
     const teamId = this.get('teamId');
     const noOwner = this.get('noOwner');
     const customFilters = omit(filtersState, 'groupId');
-    const status = [STATE_STATUS.QUEUED, STATE_STATUS.STARTED].join(',');
+    const state = states.filterByDone().pluck('id').join(',');
+    const doneState = states.filterByDone(true).pluck('id').join(',');
```

Comment:
For reference, this is the reason why changed `states.filterByDone()` away from doing `this.reset()`.